pkg/scheduler/core/generic_scheduler.go

```diff
@@ -1087,7 +1087,7 @@ func selectVictimsOnNode(
 	fitPredicates map[string]predicates.FitPredicate,
 	queue internalqueue.SchedulingQueue,
 	pdbs []*policy.PodDisruptionBudget,
-) ([]*v1.Pod, int, bool) {
+) ([]*v1.Pod, int32, bool) {
```

Comment:
is this change necessary? I prefer we just change the API

Reply:
It wouldn't be safe to return an int then cast it to int32
https://github.com/kubernetes/kubernetes/blob/84fe3db5cf58bf0fc8ff792b885465ceaf70a435/pkg/scheduler/core/generic_scheduler.go#L1014-L1021

should we change `schedulerapi.Victims.NumPDBViolations` to int64 then?

Reply:
changing NumPDBViolations to int64 will solve this issue.